duckduckgo/iOS
# Upload App Store metadata only after uploading binary

Task/Issue URL: https://app.asana.com/0/414235014887631/1203772345420825/f
CC: @jaceklyp 

**Description**:
Breaking down deliver action into two parts: first upload the binary, then attempt uploading metadata.
If the latter fails, print lengthy explanation on why it could happen (macOS app in Pending Developer Release)
and how to mitigate the problem.



**Steps to test this PR**:
1. Ensure there’s no macOS build in Pending Developer Release.
1. Run `bundle exec fastlane release_appstore` locally and verify that nothing breaks, a build is uploaded to App Store Connect and metadata is synced.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
